addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -135,18 +135,20 @@ ul.o_checklist {
             display: block;
             height: $o-checklist-before-size;
             width: $o-checklist-before-size;
-            margin-top: 4px;
+            top: 4px;
             border: 1px solid;
             text-align: center;
             cursor: pointer;
         }
         &.o_checked {
             text-decoration: line-through;
-            &::after {
+            &::before {
                 content: "✓";
-                position: absolute;
-                left: - ($o-checklist-margin-left - $o-checklist-checkmark-width);
-                top: +1px;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                padding-left: 1px #{"/*rtl:ignore*/"};
+                padding-top: 1px;
```

Comment:
Could you please check in mobile ? We used to have issues with checkboxes positioning in mobile and I fear this chang might have broken it.

Reply:
I've checked and it work.